Bottom: 92b04c3efafbb334964a5e1bb9a3fb439351cc0e
Top:    601b835a5d67a1d0e9d56fb0b23cfb46c058dd7b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-21 11:51:10 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
